content/rs/clusters/logging/log-security.md

```diff
@@ -16,48 +16,68 @@ aliases: [
+Redis Enterprise uses the default `logrotate` daemon to schedule rotation of logs stored on the operating system. You can find the log rotation configuration file at `$pkgconfdir/logrotate.conf` as of Redis Enterprise Software version 7.2 (`pkgconfdir` is `/opt/redislabs/config` by default, but can be changed in a custom installation).
 
-By default the log rotation should occur on a daily basis. We recommend that you send log files to a remote logging server so that they can be more effectively maintained.
+By default, log rotation occurs every five minutes or when logs exceed 200 MB. We recommend sending log files to a remote logging server so you can maintain them more effectively.
 
-The below log rotation policy is enabled by default with Redis Enterprise but can be modified to meet your needs.
+The following log rotation policy is enabled by default in Redis Enterprise Software, but you can modify it as needed.
 
 ```sh
-/var/opt/redislabs/log/*.log
-{ 
-  daily
-  missingok
-  copytruncate
-  rotate 7
-  compress
-  notifempty
+/var/opt/redislabs/log/*.log {
+    su ${osuser} ${osgroup}
+    size 200M
+    missingok
+    copytruncate
+    # 2000 is logrotate's way of saying 'infinite'
+    rotate 2000
+    maxage 7
+    compress
+    notifempty
+    nodateext
+    nosharedscripts
+    prerotate
+        # copy cluster_wd log to another file that will have longer retention
+        if [ "\$1" = "/var/opt/redislabs/log/cluster_wd.log" ]; then
+        	cp -p /var/opt/redislabs/log/cluster_wd.log /var/opt/redislabs/log/cluster_wd.log.long_retention
+        fi
+    endscript
+}
+/var/opt/redislabs/log/cluster_wd.log.long_retention {
+    su ${osuser} ${osgroup}
+    daily
+    missingok
+    copytruncate
+    rotate 30
+    compress
+    notifempty
+    nodateext
 }
 ```
 
-Below describes what the log rotation this configuration policy puts into effect.
+- `/var/opt/redislabs/log/*.log` - `logrotate` checks the files under directory `/var/opt/redislabs/log/` and rotates any files that end with the extension `.log`.
 
-- `/var/opt/redislabs/log/*.log` - When logrotate runs it checks the files under directory `/var/opt/redislabs/log/` and rotates any files that end with the extension .log.
+- `size 200M` - Rotate log files that exceed 200 MB.
 
-- Daily - The interval is set to daily.
+- `daily` - The interval is set to daily.
 
-- Missingok - If there are missing logfiles don't do anything.
+- `missingok` - If there are missing log files, do nothing.
 
-- Copytruncate - Truncate the original log file to zero sizes after creating a copy.
+- `copytruncate` - Truncate the original log file to zero sizes after creating a copy.
 
-- rotate 7 - Keep 7 log files and delete the rest.
+- `rotate 30` - Keep 30 log files and delete the rest.
 
-- compress - gzip log files.
+- `compress` - gzip log files.
```

Comment:
I think there should be a section for `maxage 7` too:
- `maxage 7` - Keep the rotated log files for 7 days.